CONTRIBUTING.md

```diff
@@ -0,0 +1,132 @@
+# [<img src="https://avatars1.githubusercontent.com/u/3771091?v=3&s=30">](https://github.com/gonum) Contribution Guidelines
+#### Introduction
+
+The Gonum project provides general purpose numerical computing and data science libraries for the Go programming language, and we would like you to join us in improving Gonum's quality and scope.
+This document is for anyone who is contributing or interested in contributing.
+Questions about Gonum or the use of its libraries can be directed to [gonum-dev](https://groups.google.com/forum/#!forum/gonum-dev).
+
+#### Table of Contents
+
+[Project Scope](#project-scope)
+
+[The Gonum Repositories](#the-gonum-repositories)
+
+[Contributing](#Contributing)
+  * [Working Together](#working-together)
+  * [Reporting Bugs](#reporting-bugs)
+  * [Suggesting Enhancements](#suggesting-enhancements)
+  * [Your First Code Contribution](#your-first-code-contribution)
+  * [Code Contribution](#code-contribution)
+  * [Code Review](#code-review)
+  * [What Can I Do to Help?](#what-can-i-do-to-help)
+  * [Style](#style)
+
+## Project Scope
+
+The purpose of the Gonum project is to provide general purpose numerical computing and data science libraries for the Go programming language.
+The libraries should aim to provide building blocks for disciplinary work and advanced algorithms.
+Code should be implemented in pure Go.
+Calls to C, Fortran, or other languages may be justified with performance considerations, but should be opt-in for users.
+Calls to assembly should be opt-out, if included.
+Code should favor readability and explicitness over cleverness.
+This makes code easy to review and verify, not only at submission, but also for users who want to understand how the algorithms work.
+Where possible, the source of algorithms should be referenced in the comments.
+
+## The Gonum Repositories
+
+Here are the current repositories for the gonum project.
+If code you want to contribute doesn't quite fit in any of them, then please start a discussion on the [mailing list](https://groups.google.com/forum/#!forum/gonum-dev).
+Code can be found at [github.com/gonum/](https://github.com/gonum/)\<repo\>, and documentation at godoc.org/github.com/gonum/\<repo\>.
+
+* [matrix](https://github.com/gonum/matrix) — Implements data containers for matrix types (Dense, Symmetric), and higher-level operations using them (SVD, Linear solve)
```

Comment:
How about `Data containers for various matrix types, and higher-level operations using them (factorizations, solution of linear systems)`